modules/lexy/2025.05.0/patches/doctest_quote_includes.patch

```diff
@@ -0,0 +1,830 @@
+diff --git a/include/lexy_ext/parse_tree_doctest.hpp b/include/lexy_ext/parse_tree_doctest.hpp
```

Comment:
Is there a chance to upstream this patch?

Reply:
The issue is with doctest's BUILD.bazel file imo.

https://github.com/doctest/doctest/blob/master/doctest/BUILD.bazel#L2

They don't specify an `includes` arg in their cc_library, so you can't include doctest/doctest.h using angle brackets. Angle brackets are used in their documentation and in most projects I have seen that use doctest.

The issue is that their BUILD.bazel file isn't in the root of the repo, so you would have to move that file up a directory in order to add the include path like this `includes ["."]`

I can try doing that. That'd be a breaking change for their bazel setup, which they haven't changed in 4 years though.

Reply:
OK, I have no strong opinion, just want to have a smaller patch file if possible.

Reply:
I created this bug report on doctest:
https://github.com/doctest/doctest/issues/927